README.md

```diff
@@ -1,2 +1,131 @@
+
+# 📖 [Portfolio_Microverse] <a name="about-project"></a>
+
+
+**[Portfolio_Microverse]** Is a responsive website project that describe my achievements.
+
+
+## 🛠 Built With <a name="built-with"></a>
+
+## Html
+## Css
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+
+- **[Present myself as a software developer]**
+<!-- - **[My Skills]** -->
+- **[My contact]**
+<!-- - **[My past projects]**
+- **[Responsiveness ]**
+- **[Desktop and Mobile version]**
+- **[Interactive hamburger menu]**
+- **[Modal pop up on clicking to see project]** -->
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+[Figma Link](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1&t=teTWva4DHx9UzJUX-0)
+
+[Live Demo Link](https://Mukaba/Portfolio_Microverse/)
+<!-- [Project requirements Link](Not yet available) -->
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy of this project and running, follow these steps.
+
+
+1. Download the zip file or clone with this command `git clone https://github.com/Mukaba/Portfolio_Microverse.git`(Make sure you have `Git installed` in your Computer if not then Install it by following [these steps](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git))
+2. `cd Portfolio_Microverse`
+3. Open it with the live server
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+👤 **Kitenge Mukaba Jean-louis**
+
+- GitHub: [@githubhandle](https://github.com/Mukaba)
+- Twitter: [@twitterhandle](https://twitter.com/JeanlouisMukaba)
+- LinkedIn: [LinkedIn](edin.com/in/kitenge-mukaba-jean-louis-71a2441bb/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+- [ ] **[Add header ]**
+- [ ] **[Add icon's profile links]**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+All contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Mukaba/Portfolio_Microverse).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+Give a star ⭐️ if you like this project !
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I'd like to thank my coding partners to support me when I got stuck and my code reviewers for their advices.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] The MIT license link is currently broken. I noticed you have the file name as `LICENSE` and when targeting it in your README file you target (./MIT.md). Kindly change that to (./LICENSE), and i believe that will solve the link problem.